spec/models/video_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+
+describe Video do
+  it { should validate_presence_of(:title) }
+  it { should validate_presence_of(:description) }
+  it { should belong_to(:category) }
+
+  it "saves" do
```

Comment:
Later on in lectures, you will see that testing this functionality is considered as testing ActiveRecord - and that has already been tested by Rails team. So in other words, this is over-testing and should not be practiced, because it slows your test suite. In general, guidance about this rule is - whatever code you have not written - you should not test it.